wandb/cli/cli.py

```diff
@@ -1408,6 +1408,13 @@ def launch_sweep(
     default=None,
     help="Path to the Dockerfile used to build the job, relative to the job's root",
 )
+@click.option(
+    "--priority",
+    "-P",
+    default=None,
+    type=click.Choice(["critical", "high", "medium", "low"]),
+    help="When --queue is passed, set the priority of the job. Higher priority jobs are run first.",
```

Comment:
Probably super obvious, but might be useful to explicitly define the priority order in the docstring?  E.g. 

"When `--queue` is passed, set the priority of the launch job.  Launch jobs with higher priority are served first.  The order of highest priority to lowest priority (from left to right)  is: Critical, high, medium, and low.